src/components/views/rooms/RoomList.tsx

```diff
@@ -332,7 +332,7 @@ export default class RoomList extends React.PureComponent<IProps, IState> {
             return p;
         }, [] as TagID[]);
 
-        const showSkeleton = tagOrder.every(tag => !this.state.sublists[tag]?.length);
+        const showSkeleton = Object.values(RoomListStore.instance.unfilteredLists).every(list => !list?.length);
```

Comment:
it took me far too long to figure out what this was even trying to accomplish:

```suggestion
        // show a skeleton UI if the user is in no rooms
        const showSkeleton = Object.values(RoomListStore.instance.unfilteredLists).every(list => !list?.length);
```